omnigibson/prims/rigid_prim.py

```diff
@@ -476,6 +476,11 @@ def mass(self, mass):
             mass (float): mass of the rigid body in kg.
         """
         self._mass_api.GetMassAttr().Set(mass)
+        # Write to dc as well if it's accessible
+        if self.dc_is_accessible:
+            rb_props = self.rigid_body_properties
+            rb_props.mass = mass
+            self._dc.set_rigid_body_properties(self._handle, rb_props)
```

Comment:
hmmm what is the relationship between these two different mass APIs? Also, does dc has a density API too that we need to incorporate?

Reply:
turned out to be redundant, removed